rotkehlchen/tests/utils/dataimport.py

```diff
@@ -2283,62 +2283,115 @@ def assert_binance_import_results(rotki: Rotkehlchen, websocket_connection: Webs
 
 
 def assert_rotki_generic_trades_import_results(rotki: Rotkehlchen, websocket_connection: WebsocketReader):  # noqa: E501
-    expected_trades = [
-        Trade(  # Sell 1875.64 USDC for 1 ETH
-            timestamp=Timestamp(1659085200),
-            location=Location.BINANCE,
-            base_asset=A_USDC,
-            quote_asset=A_ETH,
-            trade_type=TradeType.SELL,
-            amount=FVal('1875.64'),
-            rate=Price(FVal('0.000533151351005523447996417222921242882429464076261969247830074001407519566654582')),
-            fee=None,
-            fee_currency=None,
-            notes='Trade USDC for ETH',
-        ), Trade(  # Buy 4.3241 BTC with 392.8870 LTC
-            timestamp=Timestamp(1659171600),
-            location=Location.KRAKEN,
-            base_asset=A_BTC,
-            quote_asset=A_LTC,
-            trade_type=TradeType.BUY,
-            amount=FVal('4.3241'),
-            rate=Price(FVal('90.8598321037903841261765454082930552022386161282116509793945560925973034851183')),
-            fee=None,
-            fee_currency=None,
-            notes='Trade LTC for BTC',
-        ), Trade(  # Sell 20 UNI for 880 DAI
-            timestamp=Timestamp(1659344400),
-            location=Location.KUCOIN,
-            base_asset=A_UNI,
-            quote_asset=A_DAI,
-            trade_type=TradeType.SELL,
-            amount=FVal('20'),
-            rate=Price(FVal('44')),
-            fee=Fee(FVal('0.1040')),
-            fee_currency=A_USD,
-            notes='Trade UNI for DAI',
-        ),
-    ]
-    with rotki.data.db.conn.read_ctx() as cursor:
-        trades = rotki.data.db.get_trades(cursor, filter_query=TradesFilterQuery.make(), has_premium=True)  # noqa: E501
-
-    assert trades == expected_trades
     websocket_connection.wait_until_messages_num(num=1, timeout=10)
     assert websocket_connection.pop_message() == {
         'type': 'progress_updates',
         'data': {
             'subtype': 'csv_import_result',
             'source_name': 'Rotki generic trades',
             'total': 5,
-            'processed': 3,
+            'processed': 4,
```

Comment:
why is there one more?

Reply:
previously, we used to handle a division by zero error before but that's not the case anymore because we directly use the bought & sold amount.

Reply:
I guess it makes sense to keep it to cover the case and see that it doesn't break the logic